src/ShallowWrapper.js

```diff
@@ -1049,10 +1049,12 @@ class ShallowWrapper {
   /**
    * Returns an HTML-like string of the shallow render for debugging purposes.
    *
+   * @param {Object} options - (Optional) Property bag of additional options.
+   * options.ignoreProps - if true, props are omitted from the string.
    * @returns {String}
    */
-  debug() {
-    return debugNodes(this.getNodes());
+  debug(options) {
```

Comment:
Since this is optional, it should be defaulted (`= {}`), so it doesn't contribute to the function's length.